.github/settings.yml

```diff
@@ -0,0 +1,119 @@
+#collaborators:
+  # - username: bkeepers
+  #   permission: push
+  # - username: hubot
+  #   permission: pull
+
+  # Note: `permission` is only valid on organization-owned repositories.
+  # The permission to grant the collaborator. Can be one of:
+  # * `pull` - can pull, but not push to or administer this repository.
+  # * `push` - can pull and push, but not administer this repository.
+  # * `admin` - can pull, push and administer this repository.
+  # * `maintain` - Recommended for project managers who need to manage the repository without access to sensitive or destructive actions.
+  # * `triage` - Recommended for contributors who need to proactively manage issues and pull requests without write access.
+
+# See https://docs.github.com/en/rest/reference/teams#add-or-update-team-repository-permissions for available options
+# teams:
+    # The permission to grant the team. Can be one of:
+    # * `pull` - can pull, but not push to or administer this repository.
+    # * `push` - can pull and push, but not administer this repository.
+    # * `admin` - can pull, push and administer this repository.
+    # * `maintain` - Recommended for project managers who need to manage the repository without access to sensitive or destructive actions.
+    # * `triage` - Recommended for contributors who need to proactively manage issues and pull requests without write access.
+  # - name: org-admins
+  #  permission: admin
+
+# branches:
+  # - name: main
+    # https://developer.github.com/v3/repos/branches/#update-branch-protection
+    # Branch Protection settings. Set to null to disable
+    # protection:
+      # Required. Require at least one approving review on a pull request, before merging. Set to null to disable.
+      # required_pull_request_reviews:
+        # The number of approvals required. (1-6)
+        # required_approving_review_count: 1
+        # Dismiss approved reviews automatically when a new commit is pushed.
+        # dismiss_stale_reviews: true
+        # Blocks merge until code owners have reviewed.
+        # require_code_owner_reviews: true
+        # Specify which users and teams can dismiss pull request reviews. Pass an empty dismissal_restrictions object to disable. User and team dismissal_restrictions are only available for organization-owned repositories. Omit this parameter for personal repositories.
+        #dismissal_restrictions:
+          # users: []
+          # teams: []
+      # Required. Require status checks to pass before merging. Set to null to disable
+      # required_status_checks:
+        # Required. Require branches to be up to date before merging.
+        # strict: true
+        # Required. The list of status checks to require in order to merge into this branch
+        # contexts: []
+      # Required. Enforce all configured restrictions for administrators. Set to true to enforce required status checks for repository administrators. Set to null to disable.
+      enforce_admins: true
```

Comment:
Does this line need to be commented out for now?

Reply:
good call!

Reply:
```suggestion
      # enforce_admins: true
```